### ISSUE TITLE
ERROR: 9.12.0 manifest unknown when upgrading

### ISSUE THREAD

**Issue by nordcart** [directus/directus#13693]
### Describe the Bug

`**********\Documents\directus>docker-compose pull
Pulling database ... done
Pulling cache    ... done
Pulling directus ... error

ERROR: for directus  manifest for directus/directus:9.12.0 not found: manifest unknown: manifest unknown
ERROR: manifest for directus/directus:9.12.0 not found: manifest unknown: manifest unknown`

### To Reproduce

docker-compose pull in cmd, trying to upgrade to 9.12.0.

9.11.0 works

### What version of Directus are you using?

9.11.0

### What version of Node.js are you using?

dont know

### What database are you using?

12

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by azrikahar** [comment]
Although there is the release tag `9.12.0`, unfortunately there is currently an issue with the release pipeline, so there are no docker image nor npm package for this version yet 👍

(technically this manifest unknown error will happen whenever we are trying to use a _docker_ tag that does not exist though!)

**Comment by meditadvisors** [comment]
indeed, had the same issue, tried both docker and npm
keen to test the long awaited flows!... ;)

**Comment by paescuj** [comment]
> Although there is the release tag `9.12.0`, unfortunately there is currently an issue with the release pipeline, so there are no docker image nor npm package for this version yet 👍

High chance that it is only a one-time error. Have you already tried to simply re-run to release workflow? 😃

**Comment by vanling** [comment]
> > Although there is the release tag `9.12.0`, unfortunately there is currently an issue with the release pipeline, so there are no docker image nor npm package for this version yet 👍
> 
> High chance that it is only a one-time error. Have you already tried to simply re-run to release workflow? 😃

There were 3 attempts, all hung on being 'out of memory' Which is annoying because as far as i can tell, memory limits are not possible to increase. Maybe increasing swap file size could help here https://github.com/pierotofy/set-swap-space

**Comment by rijkvanzanten** [comment]
We currently think it's because the app codebase itself got slightly too big for rollup to build under the default memory limits in GH actions. Not seeing the same behavior locally or when increasing the memory. Hopefully the following will resolve it :)

> Error: JavaScript heap out of memory
> 
> As Rollup needs to keep all module information in memory simultaneously to be able to analyze relevant side effects for tree-shaking, it is possible that bundling large projects reaches Node's memory limit. If this happens, it can help to increase this limit by running Rollup via
> 
> `node --max-old-space-size=8192 node_modules/rollup/bin/rollup -c`
increasing --max-old-space-size as needed. Note that this number can safely surpass your available physical memory. In that case, Node will start paging memory to disk as needed.

TIL, but yeah very unfortunate that happens in such a much anticipated release 😢

**Comment by vanling** [comment]
Adding the following on the correct step in https://github.com/directus/directus/blob/main/.github/workflows/release.yml
```
env:
           NODE_OPTIONS: --max-old-space-size=8192
```
found on: https://github.com/nodejs/node/pull/39690/commits/a648ed336ba272b8b18501b1c6ef30efb6d10647

**Comment by vanling** [comment]
Haha i saw it ;) and i was spamming actions page. 

> I'm wondering if there's difference / upside/downside to specifying it globally vs per step 🤔
Are some of those node processes running in parallel? If yes, it might affect performance? I am not 100% sure if it "reserves" that 8k on memory.

**Comment by rijkvanzanten** [comment]
>  I am not 100% sure if it "reserves" that 8k on memory for every process.

AFAIK the max old space size sets Node's upper limit, but doesn't actually reserve anything. You can also set it to a number higher than the actual RAM available, and it'll start using swap files automagically if i'm not mistaken

**Comment by vanling** [comment]
yes, everyone happy if it works and else ... fail fast! and try again ^^

**Comment by rijkvanzanten** [comment]
<img width="281" alt="CleanShot 2022-06-03 at 10 52 50@2x" src="https://user-images.githubusercontent.com/9141017/171878761-80efa028-5f0f-4081-8e8e-d0516b7c049e.png">

Cool cool! Should be smooth sailing from here. It'll take a minute for the docker images to build, but at least the main bottleneck was resolved 🙂
